integration-tests/utils/integration-utils.js

```diff
@@ -45,13 +45,10 @@ const getAnimatedDrawer = textContains => {
   return `//*[@role="dialog" and @aria-hidden="false"][contains(normalize-space(.), "${textContains}") or contains(@aria-label,"${textContains}") or @aria-labelledby=//*[contains(normalize-space(.),"${textContains}")]]`
 }
 
-const clickable = ({ text, textContains, isDescendant = false }) => {
-  const base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])`
-  return getClickablePath(base, text, textContains, isDescendant)
-}
-
-const enabledClickable = ({ text, textContains, isDescendant = false }) => {
-  const base = `(//a[@aria-disabled="false"] | //button[@aria-disabled="false"] | //*[@role="button" and @aria-disabled="false"] | //*[@role="link" and @aria-disabled="false"] | //*[@role="combobox" and @aria-disabled="false"] | //*[@role="option" and @aria-disabled="false"])`
+// Note: isEnabled is not supported for native anchor and button elements.
+const clickable = ({ text, textContains, isDescendant = false, isEnabled = true }) => {
+  const checkEnabled = isEnabled !== undefined ? (isEnabled === false ? ` and @aria-disabled="${!isEnabled}"` : ` and @aria-disabled!="${isEnabled}"`) : ''
+  const base = `(//a | //button | //*[@role="button"${checkEnabled}] | //*[@role="link"${checkEnabled}] | //*[@role="combobox"${checkEnabled}] | //*[@role="option"${checkEnabled}])`
```

Comment:
No need to change this, we have spent a lot of time on it plus testing.
This is the xpath I was suggesting in slack:

```javascript
const checkEnabled = isEnabled === false ?  '[@aria-disabled="true"]' : '[not(@aria-disabled="true")]' 
const base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])${checkEnabled}`
```

Using `not` means the `@aria-disabled` attribute can be missing and it will still resolve the element.